doc/nrf/gs_programming.rst

```diff
@@ -104,8 +111,39 @@ it might return an error indicating a project load failure. For example::
 	The project file <filepath> is invalid.
 	The reported error is 'solution load command failed (1)'
 
-This issue might be caused by a variety of problems, such as incorrectly specified
-project file paths.
-SES helps you to identify the source of the issue by providing a text output
-with detailed information about the error. Make sure to click **OK** on the
-error pop-up message and then inspect the text output in SES.
+This issue might be caused by a variety of problems, such as incorrectly specified project file paths.
+SES helps you to identify the source of the issue by providing a text output with detailed information about the error.
+Make sure to click **OK** on the error pop-up message and then inspect the text output in SES.
+
+Missing executables
+===================
+
+On Windows and Linux, SES uses the PATH variable to find executables.
+If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
+If it is installed, add its location to the PATH variable.
+
+For some tools, you can explicitly specify the location under **Tools -> Options** (select the **nRF Connect** tab).
+
+  .. _ses_options_figure:
+
+  .. figure:: images/ses_options.png
+     :alt: nRF Connect SDK options in SES
+
+     nRF Connect SDK options in SES
+
+Setup on macOS
+==============
+
+On macOS, the global PATH variable is used only if you start SES from the command line.
+If you start SES by running the file :file:`bin/emStudio`, the global PATH is not used, and you must specify the path to all executables under **Tools -> Options** (select the **nRF Connect** tab, see :ref:`ses_options_figure`).
+
+In addition, specify the path to the west tool as additional CMake option, replacing *path_to_west* with the path to the west executable:
+
+.. parsed-literal::
+   :class: highlight
+
+   -D WEST=\ *path_to_west*
```

Comment:
I don't think you need a space after the `-D`? and why is there a backslash?

I think this should be something like: `-DWEST=path/to/west`

Reply:
Also, @sigvartmh could we give a reasonable default for the path? Typically if you've installed Python 3 with brew, this should be `/usr/local/bin/west`

Reply:
The space after the -D is optional - shouldn't hurt to have it.
We need the backslash space to display path_to_west in italics (which shows it's a variable that must be replaced).

Reply:
I agree a reasonable default would be  `/usr/local/bin/west` as that would be the default path for most users of `west` when installed through *pip*

Reply:
added